### PR TITLE
feat: [#1235] enhance right side panel

### PR DESCRIPTION
**Pull request by jarekzet**
Changes due to #1235 

image

### CODE REVIEW

**Comment by jarekzet** [comment]
Deleted hover animations, added new hover effects and added modifications to elements heights.

**Review by Marcelinna — APPROVED** [review]

File: ui/apps/ui/src/app/components/articles/articles.component.ts [hunk 1]
```diff
@@ -16,27 +16,10 @@ export class ArticlesComponent {
...
-      id: '2',
-      title: 'Effective Data Reuse',
-      description: 'Documentation and Metadata Refinement',
-      url: `${
-        this._configService.get().knowledge_hub_url
-      }/articles/article_03.html`,
+      title: 'EOSC Knowledge Hub',
+      description:
+        'Discover training materials, courses and learning paths supporting open science and FAIR sharing ',
+      url: `https://beta.knowledge-hub.eosc-portal.eu/ `,
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/components/articles/articles.component.ts:22]: Do we want to redirect always to the beta knowledge hub?

File: ui/apps/ui/src/app/pages/search-page/search-page.component.html [hunk 2]
```diff
@@ -68,10 +68,43 @@
...
-          style="border-left: 1px solid #d9dee2; padding-left: 20px"
+          class="col-sm-3 col-12 right-column pl-0"
         >
           <ess-articles></ess-articles>
+
+          <div class="right-links-wrapper">
+            <h4>Articles</h4>
+
+            <div class="link-box">
+              <a href="https://beta.knowledge-hub.eosc-portal.eu/articles/article_01.html">
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/pages/search-page/search-page.component.html:79]: the same question. I assume that currently there is no production version of that site, but shouldn't it be dynamically assigned?

File: ui/apps/ui/src/app/components/articles/articles.component.ts (hunk 1 above)
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/components/articles/articles.component.ts:22]: Please replace that with a variable `KNOWLEDGE_HUB_URL`: https://github.com/cyfronet-fid/eosc-search-service/blob/eb3bbd0b19bcc978eb949794a75cd251b49c701b/README.md?plain=1#L185

Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/components/articles/articles.component.ts:22]: Please replace that with a variable `KNOWLEDGE_HUB_URL`: https://github.com/cyfronet-fid/eosc-search-service/blob/eb3bbd0b19bcc978eb949794a75cd251b49c701b/README.md?plain=1#L185

File: ui/apps/ui/src/app/pages/search-page/search-page.component.html (hunk 2 above)
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/pages/search-page/search-page.component.html:79]: Construct a link using `KNOWLEDGE_HUB_URL`: https://github.com/cyfronet-fid/eosc-search-service/blob/eb3bbd0b19bcc978eb949794a75cd251b49c701b/README.md?plain=1#L185

File: ui/apps/ui/src/app/pages/search-page/search-page.component.html [hunk 3]
```diff
@@ -68,10 +68,43 @@
...
+                <div class="link-title">Expose in EOSC</div>
+                <div class="link-desc">How to expose your research in EOSC</div>
+              </a>
+            </div>
+
+            <div class="link-box">
+              <a href="https://beta.knowledge-hub.eosc-portal.eu/articles/article_02.html">
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/pages/search-page/search-page.component.html:86]: Construct a link using `KNOWLEDGE_HUB_URL`: https://github.com/cyfronet-fid/eosc-search-service/blob/eb3bbd0b19bcc978eb949794a75cd251b49c701b/README.md?plain=1#L185

File: ui/apps/ui/src/app/pages/search-page/search-page.component.html [hunk 4]
```diff
@@ -68,10 +68,43 @@
...
+                <div class="link-title">Annotate research</div>
+                <div class="link-desc">How to annotate your research</div>
+              </a>
+            </div>
+
+            <div class="link-box">
+              <a href="https://beta.knowledge-hub.eosc-portal.eu/articles/article_03.html">
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/pages/search-page/search-page.component.html:93]: Construct a link using `KNOWLEDGE_HUB_URL`: https://github.com/cyfronet-fid/eosc-search-service/blob/eb3bbd0b19bcc978eb949794a75cd251b49c701b/README.md?plain=1#L185

File: ui/apps/ui/src/app/pages/search-page/search-page.component.html [hunk 5]
```diff
@@ -68,10 +68,43 @@
...
+                <div class="link-title">Useful services </div>
+                <div class="link-desc">What useful services you can find in EOSC Catalog</div>
+              </a>
+            </div>
+
+          </div>
+
+          <a href="https://beta.marketplace.eosc-portal.eu/user_documentation" class="right-media-link">
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/pages/search-page/search-page.component.html:101]: construct a link using MARKETPLACE_BASE_URL
https://github.com/cyfronet-fid/eosc-search-service/blob/eb3bbd0b19bcc978eb949794a75cd251b49c701b/README.md?plain=1#L179

**Review by Michal-Kolomanski — CHANGES_REQUESTED** [review]
Link needs to be dynamically assigned

**Comment by Marcelinna** [comment]
@maria-j-k Please change article name similar to beta
image

**Review by Michal-Kolomanski — APPROVED** [review]

**Pull request merged**